### ISSUE TITLE
Unknown option: "--allow_remote_gui_rpc"

### ISSUE THREAD

**Issue by hemna** [BOINC/boinc-client-docker#73]
I wrote a docker-compose.yml file to stand up boinc and I get the following output.

```
waboring@hpvirt1:~/docker/boinc: $  docker-compose up
Starting boinc_boinc_1 ... done
Attaching to boinc_boinc_1
boinc_1  | Unknown option: "--allow_remote_gui_rpc"
boinc_1  | The command-line options for /usr/bin/boinc are intended for debugging.
boinc_1  | The recommended command-line interface is a separate program,'boinccmd'.
boinc_1  | Run boinccmd in the same directory as /usr/bin/boinc.
boinc_1  |
boinc_1  | Usage: /usr/bin/boinc [options]
boinc_1  |     --abort_jobs_on_exit           when client exits, abort and report jobs
boinc_1  |     --allow_remote_gui_rpc         allow remote GUI RPC connections
```

my docker-compose.yml file
```
waboring@hpvirt1:~/docker/boinc: $  cat docker-compose.yml
version: "3"
services:
  boinc:
    image: boinc/client
    volumes:
        - /mnt/nfs-export/docker/boinc:/var/lib/boinc
    environment:
        - BOINC_GUI_RPC_PASSWORD="123"
        - BOINC_CMD_LINE_OPTIONS="--allow_remote_gui_rpc"
    network_mode: "host"
    pid: "host"
```

**Comment by adamradocz** [comment]
[Here](https://github.com/BOINC/boinc-client-docker#docker-compose)'s a template for docker-compose. Try that and get back what happened.

**Comment by RichardHaselgrove** [comment]
What age/version of boinc were you trying to use? This facility was added to boinc in April 2010 by https://github.com/BOINC/boinc/commit/7a4edbafab6a653b5791344fd86eac0aca5f058a. The checking notes say

> - client: "allow_remote_gui_rpc" is available via cc_config.xml as well as command-line.

**Comment by adamradocz** [comment]
The latest. 7.16.1

**Comment by RichardHaselgrove** [comment]
Actually, the release process is up to 7.16.5 now.

7.16.1 had a bug, https://github.com/BOINC/boinc/issues/3247 and https://github.com/BOINC/boinc/pull/3249, which could have caused this.

**Comment by adamradocz** [comment]
I use the same image and it works fine.

**Comment by adamradocz** [comment]
I close this issue. If you still have this problem feel free to reopen it.

**Comment by pataquets** [comment]
@hemna : Just deployed BOINC on my PC today and had the same issue. Turned out to be the double quotes! I realized it when adding multiple command line switches and quotes were being printed only on first and last of them in the error message.
Try: `BOINC_CMD_LINE_OPTIONS=--allow_remote_gui_rpc` (without the double quotes). Worked for me.
Ubuntu Bionic, Docker 20.10.13, docker-compose 1.25.5.
cc: @adamradocz FYI
